test/images/agnhost/VERSION

```diff
@@ -1 +1 @@
-2.1
+2.2
```

Comment:
Oh the joy of serializing PR's that bump this image.  I have already built/pushed https://github.com/kubernetes/kubernetes/pull/79142 which is 2.2, and  https://github.com/kubernetes/kubernetes/pull/76838 is pending as 2.3, can you bump this to 2.4?